client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -505,6 +519,7 @@ const Progress: React.FC<IProgressProps> = ({
       )}
       {jurisdictionDiscrepanciesId && (
         <JurisdictionDiscrepancies
+          discrepancies={discrepancyQuery.data!}
```

Comment:
What does the `!` do?

Reply:
It's a non-null assertion: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Our lint rules outlaw it in VxSuite (I don't quite know the reasoning behind this, because we built a helper, `assertDefined`, to do the same thing)